aspnetcore/fundamentals/localization/sample/Resources/Views/Book/Index.resx

```diff
@@ -198,8 +198,8 @@
   <data name="Run &amp; Deploy" xml:space="preserve">
     <value>Déployer &amp; Exécuter</value>
   </data>
-  <data name="Run commands in your project.json" xml:space="preserve">
-    <value>Exécuter des commandes dans votre project.json</value>
+  <data name="Run commands in your bash" xml:space="preserve">
```

Comment:
Lots of the sentences  don't make sense. I could probably just remove all those lines. that referred to project.json. This is just to demo how to localize strings so the actual string doesn't really matter. It doesn't need to make sense. There is a big array of strings to localize and the sample only uses a few of them.